vgstation13.dme

```diff
@@ -272,7 +281,6 @@
 #include "code\game\shuttle_engines.dm"
 #include "code\game\skincmd.dm"
 #include "code\game\smoothwall.dm"
-#include "code\game\socket_talk.dm"
```

Comment:
This is intended, no fucking clue about the other ones.

Probably left DM open to fix includes and then ruined it.